File: app/src/main/java/OnTrackBus/com/Rutas.java
package OnTrackBus.com;

import java.util.ArrayList;

public class Rutas {

    private String NombreDeRuta;


    //cercania de ruta
    private Double latuser;
    private Double lnguser;



    //elementos para marcadores
    private Double latitud;
    private Double longitud;
    private String tittle;
    private String snnipet;


    public Rutas(){

    }

    public String getNombreDeRuta() {
        return NombreDeRuta;
    }

    public void setNombreDeRuta(String nombreDeRuta) {
        NombreDeRuta = nombreDeRuta;
    }


    //elementos marcador

    public Double getLatitud() {
        return latitud;
    }

    public void setLatitud(Double latitud) {
        this.latitud = latitud;
    }

    public Double getLongitud() {
        return longitud;
    }

    public void setLongitud(Double longitud) {
        this.longitud = longitud;
    }


    public String getTittle() {
        return tittle;
    }

    public void setTittle(String tittle) {
        this.tittle = tittle;
    }

    public String getSnnipet() {
        return snnipet;
    }

    public void setSnnipet(String snnipet) {
        this.snnipet = snnipet;
    }


    //elementos cercania usuario
    public Double getLatuser() {
        return latuser;
    }

    public void setLatuser(Double latuser) {
        this.latuser = latuser;
    }

    public Double getLnguser() {
        return lnguser;
    }

    public void setLnguser(Double lnguser) {
        this.lnguser = lnguser;
    }
}

File: app/src/main/java/com/android/ontrackbus/Models/Rutas.kt
package com.android.ontrackbus.Models

public class Rutas {

    private var NombreDeRuta: String? = null


    //cercania de ruta
    private var latuser: Double? = null
    private var lnguser: Double? = null


    //elementos para marcadores
    private var latitud: Double? = null
    private var longitud: Double? = null
    private var tittle: String? = null
    private var snnipet: String? = null


    fun Rutas() {}

    fun getNombreDeRuta(): String? {
        return NombreDeRuta
    }

    fun setNombreDeRuta(nombreDeRuta: String?) {
        NombreDeRuta = nombreDeRuta
    }


    //elementos marcador

    //elementos marcador
    fun getLatitud(): Double? {
        return latitud
    }

    fun setLatitud(latitud: Double?) {
        this.latitud = latitud
    }

    fun getLongitud(): Double? {
        return longitud
    }

    fun setLongitud(longitud: Double?) {
        this.longitud = longitud
    }


    fun getTittle(): String? {
        return tittle
    }

    fun setTittle(tittle: String?) {
        this.tittle = tittle
    }

    fun getSnnipet(): String? {
        return snnipet
    }

    fun setSnnipet(snnipet: String?) {
        this.snnipet = snnipet
    }


    //elementos cercania usuario
    fun getLatuser(): Double? {
        return latuser
    }

    fun setLatuser(latuser: Double?) {
        this.latuser = latuser
    }

    fun getLnguser(): Double? {
        return lnguser
    }

    fun setLnguser(lnguser: Double?) {
        this.lnguser = lnguser
    }
}